MicroProfile-Metrics/tck-runner/pom.xml

```diff
@@ -147,8 +150,8 @@
             </activation>
             <properties>
                 <!-- Metrics Dependencies -->
-                <microprofile.metrics.version>2.0.1</microprofile.metrics.version>
-                <microprofile.metrics.tck.version>2.0.1</microprofile.metrics.tck.version>
+                <microprofile.metrics.version>2.2.payara-p1</microprofile.metrics.version>
+                <microprofile.metrics.tck.version>2.2.payara-p1</microprofile.metrics.tck.version>
```

Comment:
This breaks 5.193. Probably the best course of action is limit this profile to 5.193, and since top level is 2.2, it should work for 5.194 and further. `mvn help:effective-pom` might help here to figure out what the versions are.